docs/self-hosted-appcircle/install-server/cloud-providers/_configure-server.mdx

```diff
@@ -0,0 +1,36 @@
+After you successfully connect to the Appcircle instance, the first thing you should do is start a system update. Although the Appcircle AMI is up-to-date, it is recommended that you perform security updates again.
```

Comment:
@burakberkkeskin We should add an overview to the section below for this page.

<img width="1103" alt="Screenshot 2024-12-30 at 22 24 11" src="https://github.com/user-attachments/assets/87c7e234-bd19-4426-a9b7-d2146d8d662a" />